src/components/th/th-styles.jsx

```diff
@@ -6,30 +6,62 @@ export default theme => {
   return {
     th: {
```

Comment:
This looks "similar" to `td`. We might want to extract the common parts and use it in both places.